docs/docs/cloud/binary-caching.md

```diff
@@ -74,4 +71,25 @@ The level of effectiveness that can be achieved with binary caching depends stro
 2. Define dependencies with protocol/interface targets instead of implementation ones, and dependency-inject implementations from the top-most targets.
 3. Split frequently-modified targets into smaller ones whose likelihood of change is lower.
 
-The above suggestions are part of the [µFeatures architecture](/guide/scale/ufeatures-architecture), which we propose as a way to structure your projects to maximize the benefits not only of binary caching but also of Xcode's capabilities.
\ No newline at end of file
+The above suggestions are part of the [µFeatures architecture](/guide/scale/ufeatures-architecture), which we propose as a way to structure your projects to maximize the benefits not only of binary caching but also of Xcode's capabilities.
+
+## Recommended setup
+
+We recommend having a CI job that **runs in every commit in the main branch** to warm the cache. This will ensure the cache always contains binaries for the changes in `main` so local and CI branch builds build incrementally upon them.
```

Comment:
```suggestion
We recommend having a CI job that **runs in every commit in the main branch** to warm the cache. This will ensure the cache always contains binaries for the changes in `main` so local and CI branch build incrementally upon them.
```